packages/editor/src/lib/hooks/useDocumentEvents.ts

```diff
@@ -21,9 +21,24 @@ export function useDocumentEvents() {
 			}
 			const mqString = `(resolution: ${window.devicePixelRatio}dppx)`
 			const media = matchMedia(mqString)
-			media.addEventListener('change', updatePixelRatio)
+			// Safari only started supporting `addEventListener('change',...) in version 14
+			// https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList/change_event
+			const safariCb = (ev: any) => {
+				if (ev.type === 'change') {
```

Comment:
Not sure if this is needed, I think we might only have a `change` event. Example from react native
https://github.com/necolas/react-native-web/blob/master/packages/react-native-web/src/exports/AccessibilityInfo/index.js#L30